tests/testthat/test-integrate_sces.R

```diff
@@ -0,0 +1,214 @@
+# generate testing data
+set.seed(1665)
+merged_sce <- sim_sce(n_cells = 300, n_genes = 100, n_empty = 0)
+batches <- c(rep("a", 100),
+             rep("b", 100),
+             rep("c", 100))
```

Comment:
So very minor, but here we are...
```suggestion
batches <- rep(c("a", "b", "c"), each = 100)
```